src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractRazorEditorTest.cs

```diff
@@ -53,6 +53,14 @@ public override async Task InitializeAsync()
         await TestServices.Editor.PlaceCaretAsync("</PageTitle>", charsOffset: 1, ControlledHangMitigatingCancellationToken);
         await TestServices.Editor.WaitForComponentClassificationAsync(ControlledHangMitigatingCancellationToken, count: 3);
 
+        TestServices.Input.Send("{ENTER}");
```

Comment:
I hate that it works... but it works

Reply:
Merging it will provide the real test, and we can see about tweaking it down too.